docs/Emerging-Issues.md

```diff
@@ -1,6 +1,17 @@
 ---
-title: Emerging Issues
```

Comment:
This shouldn't have been removed for the formatting structure

Reply:
Oops the nav order is gone too.